pkg/kube/inject/testdata/webhook/TestWebhookInject_validationOrder.patch

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "op": "add",
+    "path": "/spec/initContainers/-",
```

Comment:
forgive my json path knowledge, but isn't this duplicated below?

Reply:
So a trailing `-` means 'append to the end of the array `/spec/initContainers`. A trailing number says 'insert at this index'.

Reply:
I'm not sure if that answers the question, though.

Reply:
oh I thought they were the same container. but one is 'init' and one is 'validation'. lgtm